### ISSUE TITLE
weight initialization at message passing module

### ISSUE THREAD

**Issue by zguo008** [cardwing/Codes-for-Lane-Detection#106]
Hi:)

May I know why you use random normal initializer at messaging passing module given the activation function is RELU.

I think using He initializer or Xavier initialzer would  be better?

Best,

ZX

**Comment by zguo008** [comment]
Another question is that, I wonder whether dropout is turned off at testing stage?

**Comment by cardwing** [comment]
As to weight initialization, you can have a try. And yes, dropout is turned off in the testing phase.

**Comment by zguo008** [comment]
Could you please tell me more on how dropout is turned off during test phase?

Based on what I observe: in lanenet merged model: Lanenet.test_inference still calls vgg_encoder, where dropout is not turned off.

So Im quite confused loll.

Thanks!

**Comment by cardwing** [comment]
Please see [this function](https://github.com/cardwing/Codes-for-Lane-Detection/blob/master/SCNN-Tensorflow/lane-detection-model/encoder_decoder_model/cnn_basenet.py#L266). In training, dropout is enabled and in testing, it is turned off.

**Comment by zguo008** [comment]
okay:) this helps!

Thanks!
